framework/tests/flowexpression/ClassLiterals.java

```diff
@@ -0,0 +1,30 @@
+package flowexpression;
+
+import testlib.flowexpression.qual.FlowExp;
+
+public class ClassLiterals {
+    static class String {}
+
+    void method(
+            @FlowExp("String.class") Object p1,
+            @FlowExp("String.class") Object p2,
+            @FlowExp("java.lang.String.class") Object p3) {
+        @FlowExp("String.class") Object l1 = p1;
+        @FlowExp("String.class") Object l2 = p2;
+        // :: error: (assignment.type.incompatible)
+        @FlowExp("String.class") Object l3 = p3;
+        // :: error: (assignment.type.incompatible)
+        @FlowExp("java.lang.String.class") Object l4 = p1;
+        // :: error: (assignment.type.incompatible)
+        @FlowExp("java.lang.String.class") Object l5 = p2;
+        @FlowExp("java.lang.String.class") Object l6 = p3;
+    }
+
+    @FlowExp("int.class") String s0;
```

Comment:
The annotations on fields s0-s3 cause the framework to crash.  It's not critical that these expressions are supported, but they shouldn't cause a crash.  (Also it should be noted in the manual that not all class literals work.)